NEWS.md

```diff
@@ -73,6 +73,8 @@ unit = "s")
 
 10. The dimensions of objects in a `list` column are now displayed, [#3671](https://github.com/Rdatatable/data.table/issues/3671). Thanks to @randomgambit for the request, and Tyson Barrett for the PR.
 
+11. New symbol in `j`, `.NGRP`, which counts the overall number of groups in the current grouping, part of [#1206](https://github.com/Rdatatable/data.table/issues/1206).
```

Comment:
"which counts the overall" - for me it sounds a little bit like something that is not a constant, but _counts_ over current groups, maybe a wording improvement?

Reply:
I don't understand what 'groups in current grouping' is supposed to say.

I read `.NGRP` as "cardinality" of group in current process step aka current subset?  Which would be nice to have indeed!

Reply:
I'll try and come up with a quick example to add.

not sure what you mean by cardinality, but that sounds like just `.N`...

in base terms, do `y=split(DT, by='grp')`, `.N` is `nrow(y[[i]])` (it changes by group `i` which also has synonym `.GRP`) and `.NGRP` is `length(y)`

Reply:
I think that is how I understood it at first, and it is very nice to have / a good add.

In my pedestrian understanding of SQL, set theory and alike 'cardinality' usually describe the count you get for 'select distinct' ie your ,NGRP here.  But if it confuses even you then it also not a good term to use here.   Maybe above just say "which counts the overall number of groups in the currently active data set" (or  "... selection") to avoid 'group' and 'grouping' ?

Reply:
Ah, understood. From my background, cardinality means "the size of a set", just not clear which set is intended 😄

Reply:
Thanks both by the way, I definitely felt the NEWS item was being half-assed but couldn't come up with anything better at the time, I'm glad you both called me out on it 😂